src/main/java/org/osc/controller/nsc/model/VirtualizationConnectorElementImpl.java

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.controller.nsc.model;
+
+import java.util.Map;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+
+import org.osc.sdk.controller.element.VirtualizationConnectorElement;
+
+public class VirtualizationConnectorElementImpl implements VirtualizationConnectorElement {
```

Comment:
add javadoc to indicate lot of the properties are not returned.

throw UnsupportedOperationException for all methods which are not implmented and returning null etc